test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
I've added those tests to this PR to reflect the current behaviour. Please leave comments on what you expect each case to be and we'll go from there?